cities-on-map.html

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+---
+<script src="https://maps.googleapis.com/maps/api/js"></script>
```

Comment:
Not copyright, but tracking issues. I decided against linking to google maps earlier when someone told about nominatim. I think its better if we stick to open projects.